README.md

```diff
@@ -24,20 +24,18 @@ With Gradle from repo.spring.io or Maven Central repositories (stable releases o
 ```groovy
 repositories {
     mavenCentral()
-    maven { url "https://repo.spring.io/milestone" }
+
+    // Milestones
+    // maven { url "https://repo.spring.io/milestone" }
 
     // Snapshots
     // maven { url "https://repo.spring.io/snapshot" }
 }
 
 dependencies {
-    //TODO change to the release artifact and comment milestone repo above when GA
-    compile "io.projectreactor:reactor-core:3.4.0-M1"
-    testCompile "io.projectreactor:reactor-test:3.4.0-M1"
-
-    // Snapshots
-    // compile "io.projectreactor:reactor-core:3.4.0-SNAPSHOT"
-    // testCompile "io.projectreactor:reactor-test:3.4.0-SNAPSHOT"
+    compile 'io.projectreactor:reactor-core:3.3.8.RELEASE'
```

Comment:
no, the master branch is currently hosting code for 3.4.0 (which is pre-release). and I'm not sure we should showcase `reactor-tools`, which is a bit more niche: it makes it look like it is required when using reactor, but it is not.

Reply:
Since it has been moved from its own repo and is now hosted in core, I find it hard to find the dependency for it. You would need to search for it on the internet which is unfortunate.

Reply:
I have reworked the README across branches and included `reactor-tools`, with a comment